### DISCUSSION TITLE
Using hebrew/Arabic language

### DISCUSSION THREAD

**Discussion by elearnonline** [supabase/supabase#8594] [General]
Hi,
trying to upload files that contain Hebrew or Arabic characters but getting an error - Failed to upload as The key contains invalid characters!
Are there any settings to allow these special characters?

Thank u

**Comment by GaryAustin1** [comment]
I'll assume you mean upload to Supabase storage...

There are no settings for characters set.   It is based on Amazon s3 allowed key (filepath in this case) names.

https://github.com/supabase/supabase/discussions/2480
Amazon s3 key name details:   https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-keys.html
